docs/docs/deployment.md

```diff
@@ -193,6 +206,53 @@ cfg := &tls.Config{
 </TabItem>
 </Tabs>
 
+### Using GPUs
+
+If the cluster is [configured for GPU usage](./getting-started/bare-metal.md#preparing-a-cluster-for-gpu-usage), Pods can use GPU devices if needed.
+
+To do so, a CDI annotation needs to be added, specifying to use the `pgpu` (passthrough GPU) mode. The `0` corresponds to the PCI device index.
```

Comment:
> The `0` corresponds to the PCI device index.

which 0? maybe move the code snippet up.

Reply:
I see your point when reading this as Markdown, but in the docs it's quite near by. I wouldn't know where to move the code snippet here.